cpg_infra/billing_aggregator/monthly_aggregate/main.py

```diff
@@ -240,6 +213,17 @@ def get_billing_data(invoice_month: str) -> DataFrame:
     logging.getLogger('google').setLevel(logging.WARNING)
     logging.getLogger('asyncio').setLevel(logging.ERROR)
     logging.getLogger('urllib3').setLevel(logging.WARNING)
-    event_loop = asyncio.new_event_loop()
+    event_loop = asyncio.get_event_loop()
+
+    invoice_month = '202304'
```

Comment:
Just as a small one, this line seems to be redundant as later on it is replaced.